src/common/redis_module/runtest.py

```diff
@@ -85,6 +85,34 @@ def testObjectTableAddAndLookup(self):
     with self.assertRaises(redis.ResponseError):
       self.redis.execute_command("RAY.OBJECT_TABLE_ADD", "object_id3", 1, "\x00hash2", "manager_id1")
 
+  def testObjectTableAddAndRemove(self):
```

Comment:
Can you also try removing a manager that does not exist? (both when there are some managers and when there are not any managers?